docs/tools/terraform/howto/migrate-from-teams-to-groups.md

```diff
@@ -5,7 +5,27 @@ sidebar_label: Migrate from teams to groups
 
 Teams in Aiven are becoming groups. [Groups](/docs/platform/howto/manage-groups) are an easier way to control access to your organization's projects and services for a group of users.
 
-To get started using organization groups, replace your existing teams with groups.
+:::important
+**Teams have been deprecated and are being migrated to groups.**
+
+- **On September 2, 2024 the Account Owners team will be removed.**
+
+    The Account Owners and super admin
+    are synced, so the removal of the Account Owners team will have no impact on your
+    operations. [Super admin](/docs/platform/concepts/orgs-units-projects#users-and-roles)
+    have full access to organizations.
```

Comment:
Nitpick: the indent is 2 spaces too large for the list paragraphs. Not too bad to fix with multiline select in VSCode. 

```suggestion
    The Account Owners and super admin
    are synced, so the removal of the Account Owners team will have no impact on your
    operations. [Super admin](/docs/platform/concepts/orgs-units-projects#users-and-roles)
    have full access to organizations.
```